<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="folderName">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>511</width>
      <height>391</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>509</width>
       <height>389</height>
      </rect>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>371</height>
       </rect>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>491</width>
         <height>371</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="folderSearch">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Folder browser</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Search in folder</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SearchBySize">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>compare by size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Compare">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Compare</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopSearch">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop search</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="statusTip">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string>Create a new search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save a new search</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>SearchBySize</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SearchBySize_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>folderSearch</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>folderSearch_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Compare</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Compare_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopSearch</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopSearch_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>folderSearch_clicked()</slot>
  <slot>SearchBySize_clicked()</slot>
  <slot>Compare_clicked()</slot>
  <slot>stopSearch_clicked()</slot>
 </slots>
</ui>
